nextflow.config

```diff
@@ -14,8 +14,8 @@ params {
   outdir            = './results'
   input             = ''
   save_reference    = false
-  protocol          = 'chromium'
-  chemistry         = 'V3'
+  protocol          = '10X3'
```

Comment:
```suggestion
  protocol          = '10XV3'
```

Reply:
Thanks, that one (10X3) actually got me yesterday 🤦